routing/record/validation.go

```diff
@@ -100,3 +112,18 @@ func CheckRecordSig(r *pb.Record, pk ci.PubKey) error {
 	}
 	return nil
 }
+
+type RecordSorter []*pb.Record
+
+func (rs RecordSorter) Less(i, j int) bool {
+	mv, _ := MoreValid(rs[j], rs[i])
```

Comment:
No, non namespaced keys are not important.  anyone publishing non namespaced keys should not expect the correct values or even for the value to be stored